test/witan/send/acceptance/model_test.clj

```diff
@@ -61,4 +70,27 @@
     (testing "checksums are unchanged for scenario"
       (is (= expected-md5s
              (into {} (for [f files]
-                        [f (-> (io/file output-dir f) (digest/md5))])))))))
+                        [f (-> (io/file output-dir f) (digest/md5))])))))
+    (let [standard-joiner-exp (load-results "data/demo/results/joiner_beta_expectations.csv")
```

Comment:
Anyone shed any insight on why this might be failing on Circle CI? The data in `data/demo/results/` should be generated by the previous test, so should then be accessable. The error CircleCI is reporting is ```ERROR in (expected-filter-by-ay11-cy2015-results) (core.clj:4739)
Uncaught exception, not in assertion.
expected: nil
  actual: clojure.lang.ExceptionInfo: Don't know how to create dataset from shape
 at clojure.core$ex_info.invokeStatic (core.clj:4739)```